cf-monitord/mon_temp.c

```diff
@@ -86,10 +86,17 @@ void MonTempInit(void)
 {
     struct stat statbuf;
 
-    if (stat("/sys/devices/virtual/thermal", &statbuf) != -1)
+    for (int i = 0; i < 4; i++)
```

Comment:
yes, it's already hardcoded elsewhere. It was to much for me to change at this point. :-)